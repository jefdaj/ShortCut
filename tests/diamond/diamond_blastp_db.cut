pcc6803 = load_faa "data/6803genes.faa"
pcc7942 = load_faa "data/7942genes.faa"
db6803 = diamond_makedb pcc6803
result = diamond_blastp_db 1.0e-50 pcc7942 db6803
