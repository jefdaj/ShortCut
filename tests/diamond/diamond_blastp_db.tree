.
|-- cache
|   `-- load
|       |-- 8aa4a9d2fc.faa -> $WORKDIR/data/6803genes.faa
|       `-- a810086108.faa -> $WORKDIR/data/7942genes.faa
|-- exprs
|   |-- diamond_blastp_db
|   |   `-- 62073b9981_c08cc19988_d16eedcd6a.bht
|   |-- diamond_makedb
|   |   `-- c14344af19.dmnd
|   |-- load_faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   `-- dee24d6c3c.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |-- num
|   |   `-- 6de785566e.num -> ../../cache/lines/...
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/...
|       `-- ec3b181d8a.str -> ../../cache/lines/...
`-- vars
    |-- db6803.dmnd -> ../exprs/diamond_makedb/c14344af19.dmnd
    |-- pcc6803.faa -> ../exprs/load_faa/dee24d6c3c.faa
    |-- pcc7942.faa -> ../exprs/load_faa/387f499ddc.faa
    `-- result -> ../exprs/diamond_blastp_db/62073b9981_c08cc19988_d16eedcd6a.bht

9 directories, 13 files
