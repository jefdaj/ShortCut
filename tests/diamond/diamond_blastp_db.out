Synpcc7942_0001	BAA16871.1	56.7	390	151	5	16	388	1	389	1.2e-114	407.1
Synpcc7942_0003	BAA16646.1	75.3	774	177	7	6	774	11	775	0.0e+00	1153.7
Synpcc7942_0004	BAA10132.1	70.3	474	139	2	21	493	23	495	1.6e-191	662.9
Synpcc7942_0005	BAA10081.1	64.9	527	178	1	7	526	8	534	1.1e-203	703.4
Synpcc7942_0009	BAA18841.1	75.4	399	98	0	1	399	1	399	4.3e-184	637.9
...

